homeassistant/components/nest/device_trigger.py

```diff
@@ -64,7 +67,7 @@ async def async_get_device_trigger_types(
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>